chapter_09_docker.asciidoc

```diff
@@ -442,17 +442,21 @@ TIP: Don't use `export` to set the 'TEST_SERVER' environment variable;
 
 ==== Making an src Folder
 
+// DAVID: FWIW it reads weirdly to me to have 'an src' rather than 'a src'.
+// It's probably because I pronounce it as 'source' rather than 'S.R.C.'
+
 When preparing a codebase for deployment,
 it's often convenient to separate out the actual source code of our production app,
 from the rest of the files that you need in the project.
 A folder called _src_ is a common convention.
 
+// DAVID: I'd expect src here and below to be in monospace.
+
 Currently, all our code is source code really, so we move everything into _src_
-(we'll be seeing some new files appearing outside _src_ shortly.footnote:[
+(we'll be seeing some new files appearing outside _src_ shortly).footnote:[
```

Comment:
thanks for this - probably not worth spending too much time on things like this tho, o'reilly employ copy-editors to go through pernickety things like this before we go to print. and they have weird rules you can't always anticipate, like `punctuation goes inside "quotes."`